src/containers/Auth/ResetPassword/ResetPasswordConfirmOTP.test.tsx

```diff
@@ -22,10 +22,10 @@ describe('<ResetPasswordConfirmOTP />', () => {
   });
   test('it should render', async () => {
     const { findByTestId } = render(wrapper);
-
     const resetPassword = await findByTestId('AuthContainer');
-    expect(resetPassword).toHaveTextContent('Reset your password');
-    expect(resetPassword).toHaveTextContent('New Password');
```

Comment:
@mdshamoon, I was not able to find the text content  "New Password" . Could you please assist me with this?